docs/edgeql/sets.rst

```diff
@@ -248,6 +248,43 @@ Use the :eql:op:`union` operator to merge two sets.
   db> select {1, 2} union {3.1, 4.4};
   {1.0, 2.0, 3.1, 4.4}
 
+Finding common members
+----------------------
+
+.. versionadded:: 3.0
+
+Use the :eql:op:`intersect` operator to find common members between two sets.
+
+.. code-block:: edgeql-repl
+
+    db> select {1, 2, 3, 4, 5} intersect {3, 4, 5, 6, 7};
+    {3, 5, 4}
+    db> select {'a', 'b', 'c', 'd', 'e'} intersect {'c', 'd', 'e', 'f', 'g'};
+    {'e', 'd', 'c'}
+
```

Comment:
It's important to explain what happens in the case of multiple copies of the same elements:
```
db> select {0, 1, 1, 1, 2, 3, 3} intersect {1, 3, 3, 3, 3, 3};
{1, 3, 3}
```
Basically, because the intersection contains only common elements, it means that the minimum number of copies will be present in the output. There's probably a nicer way of phrasing this, though.